scripts/utils.sh

```diff
@@ -14,11 +14,48 @@ function url_reachable() {
     return $?
 }
 
+function search_for_next_free_port() {
+    service=$1
+    namespace=$2
+    port=$3
+    delete_xinetd_files_by_substr $service:$namespace:
```

Comment:
can u please explain the logic of delete_xinetd_files_by_substr

Reply:
i mean why it is part of search_for_next_free_port

Reply:
u are right thee is no reason it will be in this function